JS/Lesson18/savas/recursion.js

```diff
@@ -100,3 +250,23 @@ const person = {
 		country: 'USA',
 	},
 };
+
+function hasProperty(obj, property) {
```

Comment:
Well done here as well :)

Reply:
It's happening slowly but I think I need a lot of repetition and practice :)